doc/howto/instances_manage.md

```diff
@@ -76,3 +76,25 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+## Rebuild an instance
+
+If you want to keep the same instance configuration, but you want to freshly start with a new instance filesystem, you have the option to rebuild it.
+
+Before rebuilding an instance, you need to stop it. Also, you can only rebuild instances that have no snapshots. The reason is that rebuilding all the snapshots will require an advanced storage driver logic per backend (e.g, ZFS doesn’t allow us to delete a dataset when there are snapshots associated to it because they are inextricably linked with the original dataset (because of the CoW nature of these snapshots)) that we chose to not implement.
+
+Once these conditions are fulfilled, you can proceed with the rebuild operation. It can be done in three different ways:
```

Comment:
```suggestion
Rebuilding is only possible for instances that do not have any snapshots.
The reason for this limitation is that rebuilding an instance would also require rebuilding all snapshots, which would need advanced logic for each storage driver (because the storage drivers use very different methods for implementing snapshots).
Therefore, if your instance has snapshots, you must delete the snapshots before you can rebuild the instance.

Stop your instance before rebuilding it. 
Then enter one of the following commands:
```